config-ui/src/hooks/useDataScopesManager.jsx

```diff
@@ -154,8 +156,8 @@ function useDataScopesManager ({ provider, blueprint, /* connection, */ settings
           newScope = projects[connection.id]?.map((p) => ({
             ...newScope,
             options: {
-              owner: p.split('/')[0],
-              repo: p.split('/')[1],
+              owner: p.value.split('/')[0],
```

Comment:
Use optional chaining check here for both owner and repo (`?`)
```suggestion
              owner: p.value?.split('/')[0],
```